index.md

```diff
@@ -1,7 +1,10 @@
 ---
 title: "Make your Data Management Plan comes true - Tips and Tricks"
```

Comment:
Can you put "Tip and Tricks" entirely in a new line? Like a subtitle, for example, or just in a new line under the first sentence. I think it will look better.

Reply:
![Screenshot from 2020-11-20 14-30-41](https://user-images.githubusercontent.com/44875756/99810987-d10d1c80-2b44-11eb-823e-e729221db9c9.png)

This is maybe to small ? :)

Reply:
Other small thing,  `Make your Data Management Plan come true` with comes singular?

Reply:
yes, too small. I don't mean in the summary, just on a second line (moving "Tips" in the same line of "and Tricks"). Something like 
"Make your Data Management Plan comes true:
Tips and Tricks".

I think "comes" is correct, but I am not sure. We should ask an English mother-tongue.

Reply:
I can will support a sub title

because data management plan is singular, I think come has to be too ... :)